genie-web/src/integTest/java/com/netflix/genie/web/data/services/impl/jpa/JpaPersistenceServiceImplJobsIntegrationTest.java

```diff
@@ -1185,8 +1186,18 @@ void canGetAgentActiveJobs() {
 
         Assertions
             .assertThat(this.service.getActiveAgentJobs())
-            .hasSize(2)
-            .containsExactlyInAnyOrder(AGENT_JOB_1, AGENT_JOB_2);
+            .hasSize(3)
+            .containsExactlyInAnyOrder(AGENT_JOB_1, AGENT_JOB_2, AGENT_JOB_3);
+    }
+
+    @Test
+    @DatabaseSetup("persistence/jobs/search.xml")
```

Comment:
One reason I broke it apart like this is so that individual tests can use their own setup file to prevent the trickle you're seeing here where you had to change the counts in other test methods cause you added this test and changed this file.

Instead you could copy this file into some other scoped file and limit it down to just what you want to test.

You don't have to do this obviously but it keeps the tests a little more isolated/focussed imo.

Reply:
Makes sense, didn't even occur to me but i should use a separate file.